certs/certs.go

```diff
@@ -83,7 +87,7 @@ func Delete(c *deis.Client, name string) error {
 	return err
 }
 
-// Attach a certificate to a domain
+// Attach adds a domain to a certificate.
```

Comment:
Unlike my other nits, I like this just the way it is, because the meaning of "attach" isn't immediately contextually obvious. Same goes for `Detatch`. 👍